src/frontend/src/locales/uk.json

```diff
@@ -59,53 +59,122 @@
                   "password": "Пароль",
                   "tip-password": "Пароль повинен мати хоча б 8 символів та містити числа та букви",
                   "tip-email": "Електронна адреса має містити @",
-                  "confirm-password":"Підтвердьте пароль",
-                  "agree" : "Я згоден з",
-                  "privacy policy" : "політикою конфіденційності",
-                  "register" : "Зареєструватися",
-                  "processing" : "Обробка...",
+                  "confirm-password": "Підтвердьте пароль",
+                  "agree": "Я згоден з",
+                  "privacy policy": "політикою конфіденційності",
+                  "register": "Зареєструватися",
+                  "processing": "Обробка...",
                   "error-nickname-exist": "Цей псевдонім вже існує, придумайте щось інше"
             },
-            "tip":{
```

Comment:
please, use explanation after word "nickname-". Do not use "-tip(1,2,...)" . it is difficult to read. if it is connected to chars and numbers, write "nickname-chars-numbers".